pds_doi_core/actions/test/check_test.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import unittest
+import os
+
+from types import SimpleNamespace
+
```

Comment:
unit tests worked. Great.

Reply:
Good to hear.  My unit test just ran too.   Also removed unused SimpleNamespace import since we don't need it anymore.

Reply:
ok, they still work.